### PR TITLE
link to STAC API Extensions site instead of listing them

### PR DESCRIPTION
**Pull request by philvarner**
**Related Issue(s):** 

- n/a

**Proposed Changes:**

1. link to STAC API Extensions site instead of listing them

**PR Checklist:**

- [X] This PR has **no** breaking changes.
- [X] This PR does not make any changes to the core spec in the `stac-spec` directory (these are included as a subtree and should be updated directly in [radiantearth/stac-spec](https://github.com/radiantearth/stac-spec))
- [X] I have added my changes to the [CHANGELOG](https://github.com/radiantearth/stac-api-spec/blob/main/CHANGELOG.md) **or** a CHANGELOG entry is not required.

### CODE REVIEW

File: core/README.md [hunk 1]
```diff
@@ -7,14 +7,6 @@
   - [Extensions](#extensions)
   - [Structuring Catalog Hierarchies](#structuring-catalog-hierarchies)
 
-- **OpenAPI specification:** [openapi.yaml](openapi.yaml) ([rendered version](https://api.stacspec.org/v1.0.0-rc.2/core)),
```
Comment (m-mohr) [inline core/README.md:10]: Is this intentional? If yes, why?

Comment (philvarner) [reply core/README.md:10]: It's not -- it's a problem with the VS Code markdown plugin. I'll fix this.

File: README.md [hunk 2]
```diff
@@ -116,9 +116,8 @@ The *[item-search](item-search)* folder contains the Item Search specification,
 cross-collection search of STAC Item objects at a `search` endpoint, as well as a number of extensions. 
 
 **Extensions:**
-The *[extensions](extensions.md) document* describes how STAC incubates new functionality, and it links to the existing 
-extensions that can be added to enrich the functionality of a STAC API. Each has an OpenAPI yaml, but some of the yaml
-documents live as fragments in the [fragments/](fragments/) folder. The official list of STAC API Extensions
+The *[extensions](extensions.md) document* describes how STAC adds new functionality
+through extensions. The official list of STAC API Extensions
 is maintained [here](https://stac-api-extensions.github.io).
```
Comment (m-mohr) [inline README.md:121]:
```suggestion
is maintained at [stac-api-extensions.github.io](https://stac-api-extensions.github.io).
```

"here" links generally are not a good practice.

File: core/README.md [hunk 3]
```diff
@@ -231,7 +231,7 @@ different conformance classes and a different set of links.
 
 ## Extensions
 
-None.
+STAC API Extensions can be found [here](https://stac-api-extensions.github.io).
```
Comment (m-mohr) [inline core/README.md:234]:
```suggestion
STAC API Extensions can be found at [stac-api-extensions.github.io](https://stac-api-extensions.github.io).
```

File: item-search/README.md [hunk 4]
```diff
@@ -297,10 +297,4 @@ the [overview](../overview.md#example-landing-page) document.
...
 ## Extensions
 
-These extensions provide additional functionality that enhances Item Search.
-
-- [Fields Extension](https://github.com/stac-api-extensions/fields/blob/main/README.md)
-- [Sort Extension](https://github.com/stac-api-extensions/sort/blob/main/README.md)
-- [Context Extension](https://github.com/stac-api-extensions/context/blob/main/README.md)
-- [Filter Extension](https://github.com/stac-api-extensions/filter/blob/main/README.md)
-- [Query Extension](https://github.com/stac-api-extensions/query/blob/main/README.md)
+STAC API Extensions can be found [here](https://stac-api-extensions.github.io).
```
Comment (m-mohr) [inline item-search/README.md:300]:
```suggestion
STAC API Extensions can be found at [stac-api-extensions.github.io](https://stac-api-extensions.github.io).
```

File: ogcapi-features/README.md [hunk 5]
```diff
@@ -396,12 +396,4 @@ as is typical with a static STAC Collection, there are no links here with rel va
...
-These extensions provide additional functionality that enhances *STAC API - Features*.
-
-- [Transaction Extension](https://github.com/stac-api-extensions/transaction/blob/main/README.md)
-- [Items and Collections API Version Extension](https://github.com/stac-api-extensions/version/blob/main/README.md)
-- [Fields Extension](https://github.com/stac-api-extensions/fields/blob/main/README.md)
-- [Sort Extension](https://github.com/stac-api-extensions/sort/blob/main/README.md)
-- [Context Extension](https://github.com/stac-api-extensions/context/blob/main/README.md)
-- [Filter Extension](https://github.com/stac-api-extensions/filter/blob/main/README.md)
-- [Query Extension](https://github.com/stac-api-extensions/query/blob/main/README.md)
+STAC API Extensions can be found [here](https://stac-api-extensions.github.io).
```
Comment (m-mohr) [inline ogcapi-features/README.md:399]:
```suggestion
STAC API Extensions can be found at [stac-api-extensions.github.io](https://stac-api-extensions.github.io).
```

**Review by m-mohr — APPROVED** [review]
Looks good, but I'd really replace the "here" links to something more concrete asn here links in the web are basically discouraged for accessibility reasons.

**Comment by philvarner** [comment]
> Looks good, but I've replaced the "here" links to something more concrete as "here" links in the web are basically discouraged for accessibility reasons.

Thanks, I just got lazy there.

**Pull request merged**